package.json

```diff
@@ -148,7 +148,7 @@
     "react-native-web": "^0.18.12",
     "react-router-dom": "^5.3.4",
     "react-test-renderer": "^18.2.0",
-    "storybook": "7.6.7",
+    "storybook": "^7.6.20",
```

Comment:
I _think_ it would be a great idea to update them too, **but** there probably will need to be more effort put into testing, since that's a lot of dependencies to update.